detox/src/logger/DetoxLogger.js

```diff
@@ -119,6 +119,25 @@ class DetoxLogger {
     this.overrideConsole();
   }
 
+  /**
+   * Closes the file descriptors to make sure that the temporary
+   * JSONL files can be moved safely to the directory with the artifacts.
+   * This safety measure is especially important for Windows OS.
+   *
+   * @async
+   * @internal
+   */
+  async close() {
```

Comment:
Hm, overall, I wonder if we could try just flushing the logger instead of closing the file streams.

Reply:
Seems not really viable. We could try to subscribe to the `drain` event or something, and try to wait, but closing looks more solid.